unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/EventsSubSection/EventsSubSectionMenu/EventsSubSectionComponentController.cs

```diff
@@ -235,4 +312,45 @@ public static void JumpInToEvent(EventFromAPIModel eventFromAPI)
         else
             Environment.i.world.teleportController.JumpIn(coords.x, coords.y, serverName, layerName);
     }
+
+    private void RequestAndLoadCategories()
+    {
+        eventsAPIApiController.GetCategories(
+            onSuccess: categoryList =>
+            {
+                List<CategoryFromAPIModel> categoriesInUse = new ();
+                foreach (CategoryFromAPIModel category in categoryList)
+                {
+                    foreach (EventFromAPIModel loadedEvent in eventsFromAPI)
+                    {
+                        if (!loadedEvent.categories.Contains(category.name))
+                            continue;
+
+                        categoriesInUse.Add(category);
+                        break;
+                    }
+                }
+
+                view.SetCategories(PlacesAndEventsCardsFactory.ConvertCategoriesResponseToToggleModel(categoriesInUse));
+            },
+            onFail: error => { Debug.LogError($"Error receiving categories from the API: {error}"); });
+    }
+
+    private static bool IsTimeInRange(string dateTime, float lowTimeValue, float highTimeValue)
+    {
+        string startTimeString = ConvertToTimeString(lowTimeValue);
+        string endTimeString = ConvertToTimeString(highTimeValue);
+
+        TimeSpan startTime = lowTimeValue < TIME_MAX_VALUE ? TimeSpan.Parse(startTimeString) : new TimeSpan(1, 0, 0, 0);
```

Comment:
For readability's sake, I would avoid using `new TimeSpan` and instead use one of its static constructors like `TimeSpan.FromSeconds` or `TimeSpan.FromDays`